### ISSUE TITLE
`@stylistic/selector-pseudo-class-parentheses-space-inside` reads a multi-line argument list by its line feeds alone

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#173]
Found while fixing [#163](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/163), which taught the rule to read a selector carrying an inline comment. It reaches a line that was written for line feeds alone, and the same line has always been there for plain CSS.

### What minimal example or steps are needed to reproduce the bug?

A pseudo-class whose arguments are broken over lines with a carriage return, or with a form feed:

```css
a:not( b\r, c ) {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/selector-pseudo-class-parentheses-space-inside": "never"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

The same answer a line feed gets. `never` leaves the whitespace in front of the closing parenthesis of a multi-line argument list alone, and a list broken with a carriage return or a form feed is a multi-line list too.

### What actually happened?

The whitespace is reported and taken away, so a list broken one way is spared and the same list broken another way is closed up.

The rule decides it here:

```js
let isParamStringMultiline = paramString.includes(`\n`)
```

and `never` reads that answer for the closing side only, so the two spellings part exactly there.

### Do you have a proposal to fix the bug?

`(/[\n\f\r]/u).test(paramString)` is the whole of it — those are the characters the syntaxes end a line on, and `findSelectorInlineComments` reads them that way already.

Worth deciding along with it: what a multi-line list is for the *opening* side. `never` reads `nextCharIsSpace` as `paramString.startsWith(" ")`, so a list opening with a line break of any kind is passed over there whatever this line says, while `always` asks for a single space at both ends and gets neither. That asymmetry is older than this and is not what makes the two line endings differ, but a reader of the fix will meet it in the same paragraph.
